src/yb/util/wait_state.h

```diff
@@ -12,6 +12,7 @@
 //
 #pragma once
 
+#include <array>
```

Comment:
@amitanandaiyer is this change needed?

Reply:
Yes. Abhinab has used an array in this file.

The compiler on my laptop complains/requires this. Not sure why others don't complain.